src/components/ProductDetails/ProductDetails.jsx

```diff
@@ -1,17 +1,46 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import ProductInfo from './ProductInformation.jsx';
+import ImageGallery from './ImageGallery.jsx';
+import Cart from './Cart.jsx';
+import axios from 'axios';
+import config from '../../../config.js';
 
 class ProductDetails extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-
+      styles: [],
+      currentStyle: {},
     }
   }
 
+  componentDidMount() {
+    axios.defaults.headers.common['Authorization'] = config.TOKEN;
+    axios.get(`https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfp/products/${this.props.id}/styles`).then((data) => {
+      // console.log('style data in Style selector',data.data.results[0])
+      this.setState({styles: data.data.results, currentStyle:data.data.results[0]})

```

Comment:
Should the current style just be the first style in the styles array, or should it be the style with 'default?: true' key value pair?

Reply:
Good catch, I've adjusted it to prioritize the 'default?' value, and then fall back to the first entry in the array if there is no default? value in the commits below